src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php

```diff
@@ -122,15 +122,12 @@ private function parseResource(ResourceMetadata $resourceMetadata, string $resou
      * Returns groups of item & collection.
      *
      * @param ResourceMetadata $resourceMetadata
-     * @param string           $resourceClass
-     * @param string[]         $visited
+     * @param string           $operationName
      *
      * @return array
      */
-    private function getGroupsForItemAndCollectionOperation(ResourceMetadata $resourceMetadata, string $resourceClass, string $operationName, array $visited = []) : array
+    private function getGroupsForItemAndCollectionOperation(ResourceMetadata $resourceMetadata, string $operationName) : array
```

Comment:
This function should be removed entirely, per previous discussion. It's only here due to failure to properly utilize the fallback mechanism, isn't it?

/cc @Simperfit @dunglas